elements/pf-select/pf-select.ts

```diff
@@ -163,11 +163,29 @@ export class PfSelect extends LitElement {
     return this.variant === 'checkbox' && !this.checkboxSelectionBadgeHidden;
   }
 
-  /**
-   * array of text content from listbox's array of selected options
-   */
-  get #valueTextArray() {
-    return this.#listbox?.selectedOptions.map(option => option.optionText || '') ?? [];
+  get #buttonLabel() {
```

Comment:
Button label **_should not_** be the placeholder or a selected value. Use `accessibleLabel` or have a way to associate the select with a `<label>` element.